db/migrate/20220322024216_add_bookmark_polymorphic_columns_again.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class AddBookmarkPolymorphicColumns < ActiveRecord::Migration[6.1]
+class AddBookmarkPolymorphicColumnsAgain < ActiveRecord::Migration[6.1]
```

Comment:
I'm not sure we need `again` here. Now that we have deleted the old migrations, technically we're only adding it once.

Reply:
Yah good point, will just rename

Reply:
Done